src/components/Histogram/HistogramComponent.tsx

```diff
@@ -194,30 +194,23 @@ export class HistogramComponent extends React.Component<WidgetProps> {
         let profilerInfo: string[] = [];
         if (this.plotData) {
             if (this.widgetStore.isMouseMoveIntoLinePlots) {
-                let numberString;
-                // Switch between standard and scientific notation
-                if (this.widgetStore.cursorX < 1e-5) {
-                    numberString = toExponential(this.widgetStore.cursorX, 5);
-                } else {
-                    numberString = toFixed(this.widgetStore.cursorX, 5);
-                }
                 const nearest = binarySearchByX(this.plotData.values, this.widgetStore.cursorX);
                 if (nearest?.point) {
-                    let valueLabel = `${nearest.point.y !== 0.5 ? nearest.point.y : 0}`;
-
+                    let xValueLabel = Math.abs(nearest.point.x) < 1e-5 ? `${toExponential(nearest.point.x, 5)}` : `${toFixed(nearest.point.x, 5)}`;
```

Comment:
One minor suggestion: we can remove `${...}` because `toExponential` and `toFixed` return string.

Reply:
corrected.

Reply:
Oh, I mean we can remove ' ${...} '

Reply:
corrected.